### PR TITLE
Implemented Nexus upload classifier parameter.

### PR DESCRIPTION
**Pull request by alper**
An issue I ran in to to day and saw that has been created as a ticket by somebody else as well here: https://github.com/fastlane/fastlane/issues/4023

### CODE REVIEW

File: fastlane/lib/fastlane/actions/nexus_upload.rb [hunk 1]
```diff
@@ -32,6 +32,11 @@ def self.upload_options(params)
         options << "-F g=#{params[:repo_group_id].shellescape}"
         options << "-F a=#{params[:repo_project_name].shellescape}"
         options << "-F v=#{params[:repo_project_version].shellescape}"
+
+        if params.fetch(:repo_classifier, ask: false)
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/nexus_upload.rb]: You should be able to just use `params[: repo_classifier]` instead. The value is optional, so the user won't be asked

Comment (alper) [inline fastlane/lib/fastlane/actions/nexus_upload.rb]: Thanks, I just learned Ruby to do this so forgive my non-idiomaticness. I just pushed the change.

**Comment by alper** [comment]
Hey, this isn't a Big PR despite danger's warning. I would even say it's fairly trivial. Could you merge this so that we can update our fastlane?

**Comment by KrauseFx** [comment]
Sorry about the danger warning, taking a look at this now 👍

**Comment by KrauseFx** [comment]
Sorry for not coming back to your earlier.

**Comment by KrauseFx** [comment]
Thank you for your contribution, this is great 👍 This will be in the next release 🚀

**Pull request merged**